### ISSUE TITLE
[BUG] Transposed mappings for Android Rule/WCAG guidance

### ISSUE THREAD

**Issue by DaveTryon** [microsoft/accessibility-insights-web#2712]
Please check whether the bug has [already been filed](https://github.com/Microsoft/accessibility-insights-windows/issues).

**Describe the bug**
In AI for Android 2020.515.2, the WCAG mappings for TouchSizeWcag and ActiveViewName are transposed. TouchSizseWcag should point to WCAG 2.5.5, and ActiveViewName should point to 1.3.1 and 3.3.2.

**To Reproduce**
1. Start AI for Android
2. Scan the test app
3. Look at the results

**Expected behavior**
The mapped links should be appropriate targets for the rule in question.

**Actual behavior**
The links are transposed

**Screenshots or .GIF**
![image](https://user-images.githubusercontent.com/45672944/82258732-3cb0cb80-990f-11ea-9a58-d523d5415237.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10 1809] (Get the version by running `winver` from the command line)
 - Accessibility Insights for Windows Version:
 - Target Application: [e.g. Chrome, Wildlife Manager]
 - Target Application Version: [e.g. 22]

**Additional context**
Priority requested -

**Comment by DaveTryon** [comment]
This was fixed in #2713 and has been validated in 2020.518.7
